frontend/src/Context/index.tsx

```diff
@@ -4,8 +4,11 @@ interface QuickstartState {
   linkSuccess: boolean;
   isItemAccess: boolean;
   isPaymentInitiation: boolean;
+  isUserTokenFlow: boolean;
```

Comment:
Can we just use `userToken != null` to mean the same thing?

Reply:
I'm using `isUserTokenFlow` to determine if we should call `/user/create`. When that call is made initially, `userToken` will always be `null` so I can't just use  `userToken != null`